janus-idp/operator
# Add Route Host and TLS configuration to Backstage CR

## Description
Added host, subdomain and TLS certificates to the OpenShift route configuration in the Backstage CRD.

## Which issue(s) does this PR fix or relate to

- Fixes #13 

## PR acceptance criteria


## How to test changes / Special notes to the reviewer
a) Deploy the operator on OpenShift cluster.
b) Deploy CR: examples/bs-route.yaml
c) Check that an OpenShift route with host test-bs-route-<cluster base URL> has been created and the page https:// test-bs-route-<cluster base URL> is accessible.
